.github/workflows/reusable-container-workflow.yaml

```diff
@@ -137,7 +137,47 @@ jobs:
           platforms: linux/amd64,linux/arm64
           build-args: |
             QEMU_CPU=max,pauth-impdef=on
+          push: false
+          tags: |
+            ${{ steps.metadata.outputs.tags }}
+          labels: ${{ steps.metadata.outputs.labels }}
+          file: ${{ matrix.dockerfile }}-${{ inputs.build_type }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          load: true
```

Comment:
Looks like multi-plaform loading does not work 🤔 https://github.com/docker/build-push-action/issues/733#issuecomment-1332515783